tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Builders/OrganisationBuilder.cs

```diff
@@ -16,9 +20,22 @@ private OrganisationBuilder()
 
         public Organisation Build()
         {
-            return new() { Name = name, OrganisationId = organisationId };
+            Organisation toBuild = new() { Name = name, OrganisationId = organisationId };
+
+            if (relatedOrganisations is not null && relatedOrganisations.Any())
+                relatedOrganisations.ToList().ForEach(ro => toBuild.RelatedOrganisations.Add(ro));
+
+            return toBuild;
         }
 
         internal static OrganisationBuilder Create() => new();
+
+        internal OrganisationBuilder WithRelatedOrganisations(List<Guid> relatedOrganisations)
+        {
+            this.relatedOrganisations = relatedOrganisations
+                                        .Select((value, index) => new { value, Name = string.Format(CultureInfo.CurrentCulture, "HealthTrust {0}", index) })
+                                        .Select(ro => new Organisation { OrganisationId = ro.value, Name = ro.Name });
```

Comment:
We could always just name the parameter `organisations` to avoid hiding the field of the same name. I don't mind this naming style though – much better than the loathsome underscore prefix for fields that some people seem to adore! 😃

One other small thing, I'd only indent the `.Select` lines by one tab stop to keep the style consistent (and probably add a blank line before the `return`.

Reply:
i'd like to keep it just as relatedOrganisations just so when we come back to it in the future we know exactly what it's on about. even though the function does tell you i think it's just more descriptive

Reply:
OK, that's fine.